packages/accounts/src/msca/plugins/multi-owner/signer.ts

```diff
@@ -0,0 +1,46 @@
+import type { SignTypedDataParams, SmartAccountSigner } from "@alchemy/aa-core";
+import { toBytes } from "viem";
+import { MultiOwnerPlugin } from "./plugin.js";
+
+export class MultiOwnerPluginSigner
+  implements SmartAccountSigner<SmartAccountSigner>
+{
+  signerType: string;
+  inner: SmartAccountSigner<any>;
+  plugin: MultiOwnerPlugin;
+
+  // TODO(moldy) -- use a struct instead of args array
+  constructor(inner: SmartAccountSigner<any>, pluginAddress: string) {
+    this.signerType = inner.signerType;
+    this.inner = inner;
+    this.plugin = new MultiOwnerPlugin(pluginAddress);
+  }
+
+  // This returns the address of the current owner, not all owners
+  getAddress: () => Promise<`0x${string}`> = () => {
+    return this.inner.getAddress();
+  };
+
+  // NOTE: signing Userops like this will not work because UOs are signed the old way...
+  // TODO(jay + moldy): work with howy to do this consistently for UOs and 1271
```

Comment:
could have a special method here called `signMessagefor1271` here and use that in the account as well. signMessage can be used to sign UOs but devs would have to call `signMessageFor1271` manually for all other cases